### ISSUE TITLE
Support sort-only field

### ISSUE THREAD

**Issue by tongwang** [burke-software/django-report-builder#204]
It'd be nice if we can have non-displaying fields for sorting purpose. For example, a model has two fields: `title`, `sort_key`.  I'd like to sort them by `sort_key`, but only display `title` field. Can't find a way to do that.

**Comment by grantjenks** [comment]
You're right. You can't current do this.

It's actually a little awkward the way Display and Sort are in the same tab. I wonder if we should move sort fields to a different tab and let them be ordered automatically in descending order.

@bufke Your thoughts?

**Comment by bufke** [comment]
Yeah we could either allow a sort only field or give the UI overall thought. If my company gets some downtime I'll have a designer give this some thought.
